### PR TITLE
Remove permission_classes from un-opinionated views to allow customization

### PR DESCRIPTION
**Pull request by AlanCoding**

### CODE REVIEW

File: ansible_base/lib/utils/views/ansible_base.py [hunk 1]
```diff
@@ -10,6 +11,8 @@
 
 
 class AnsibleBaseView(APIView):
+    permission_classes = [permissions.IsAuthenticated]
```
Comment (AlanCoding) [inline ansible_base/lib/utils/views/ansible_base.py]: @john-westcott-iv @relrod @slemrmartin Maybe the better solution here would be to _not_ set this at all, with the intent that it'll just use the default permission class from DRF? In practice, we would do this for test_app.

All the real apps using DAB will have something concrete set for this. By not putting it here, apps can customize the permission class (to its own default) without changing the base view.

Comment (slemrmartin) [reply ansible_base/lib/utils/views/ansible_base.py]: I agree, it sounds more generic

**Review by john-westcott-iv — APPROVED** [review]

File: ansible_base/lib/utils/views/ansible_base.py (hunk 1 above)
Comment (john-westcott-iv) [reply ansible_base/lib/utils/views/ansible_base.py]: I agree with removing this from ansible-base but added an approval on this.

Comment (AlanCoding) [reply ansible_base/lib/utils/views/ansible_base.py]: Thanks. I will go ahead and make this change.

**Comment by AlanCoding** [comment]
Updated title of PR to reflect the latest change.

Confirmed that /api/v1/authenticators/ is still working in the browser.

**Review by slemrmartin — APPROVED** [review]

**Pull request merged**